cmis_web_proxy/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Cmis Web Proxy',
+    'summary': """
+        Odoo as proxy server for your cmis requests.""",
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV',
+    'website': 'https://alfodoo.org/',
+    'depends': [
+        'cmis_web'
+    ],
+    'data': [
+        'views/cmis_backend.xml',
+        'views/cmis_web_proxy.xml'
+    ],
+    'external_dependencies': {
+        'python': ['requests'],
```

Comment:
`'cmislib'` is missing in the list of external dependencies

Reply:
@lmignon The external dependency should be on the cmis module, not this one.